kokoro/config/test/ops_agent/mantic_x86_64.gcl

```diff
@@ -4,6 +4,5 @@ import '../image_lists.gcl' as image_lists
 config build = common.ops_agent_test {
   params {
     platforms = image_lists.mantic_x86_64.presubmit
```

Comment:
I see all the arm64 gcl files still keep the arch field and only this one removes it. Why is it?

Reply:
For the x86 gcls, if we omit the `arch` parameter, the final configuration will use the `arch = 'amd64'` value from `common.gcl` and then pick the zones. For arm64 gcls we need to keep the `arch` parameter in each gcl, so that it will override the default value in `common.gcl`.